c/httpserver.c

```diff
@@ -3476,6 +3481,8 @@ static int handleHttpService(HttpServer *server,
     } else {
       respondWithAuthError(response, &authResponse);
     }
+  } else if (!authorized) {
+    respondWithError(response, HTTP_STATUS_FORBIDDEN, "Forbidden");
```

Comment:
How does this affect the endpoints which are open to everyone? Do we want authorization be ignored the same way authentication is when `SERVICE_AUTH_FLAG_OPTIONAL`?